.github/workflows/manual-full-build.yaml

```diff
@@ -46,41 +65,65 @@ jobs:
           echo ::set-output name=ocsdevlatest::${CORE_OCS_DEV_TAG}
 
       - name: Login to DockerHub Registry
-        run: echo ${{ secrets.GHACTIONSDOCKERHUB }} | docker login -u ${{ secrets.GHACTIONSDOCKERHUBNAME }} --password-stdin
+        run: echo ${{ secrets.GHACTIONSDOCKERHUB }} | podman login -u ${{ secrets.GHACTIONSDOCKERHUBNAME }} --password-stdin
+
+      - name: Login to Quay Registry
+        run: echo ${{ secrets.GHACTIONQUAYTOKEN }} | podman login quay.io -u ${{ secrets.GHACTIONQUAYNAME }} --password-stdin
+
+      - name: Build Builder Images
+        run: |
+          buildah build \
+            -f src/deploy/NVA_build/builder.Dockerfile \
+            --platform=$PLATFORMS \
+            --manifest localhost/noobaa-builder
+          echo "ℹ️ Inspect noobaa-builder manifest"
+          skopeo inspect --raw containers-storage:localhost/noobaa-builder
+
+      - name: Build Base Images
+        run: |
+          buildah build \
+            -f src/deploy/NVA_build/Base.Dockerfile \
+            --platform=$PLATFORMS \
+            --manifest localhost/noobaa-base
+          echo "ℹ️ Inspect noobaa-base manifest"
+          skopeo inspect --raw containers-storage:localhost/noobaa-base
+
+      - name: Build NooBaa Images
+        run: |
+          buildah build \
+            -f src/deploy/NVA_build/NooBaa.Dockerfile \
+            --build-arg GIT_COMMIT=$GIT_COMMIT \
+            --platform=$PLATFORMS \
+            --manifest localhost/noobaa
+          echo "ℹ️ Inspect noobaa manifest"
+          skopeo inspect --raw containers-storage:localhost/noobaa
 
-      - name: Build & Push Docker Images to DockerHub
+      - name: Push Docker Images to DockerHub
```

Comment:
@baum let's remove the push steps. for now we can use this action as a test for our build process and make sure that arm builds are not broken. later on, we can use it to publish multiarch builds, after we have testing pipeline for that